include/openssl/ssl.h

```diff
@@ -169,17 +169,23 @@ extern "C" {
  * The following cipher list is used by default. It also is substituted when
  * an application-defined cipher list string starts with 'DEFAULT'.
  * This applies to ciphersuites for TLSv1.2 and below.
+ * DEPRECATED IN 3.0.0, in favor of OSSL_default_cipher_list()
  */
-# define SSL_DEFAULT_CIPHER_LIST "ALL:!COMPLEMENTOFDEFAULT:!eNULL"
-/* This is the default set of TLSv1.3 ciphersuites */
-# if !defined(OPENSSL_NO_CHACHA) && !defined(OPENSSL_NO_POLY1305)
-#  define TLS_DEFAULT_CIPHERSUITES "TLS_AES_256_GCM_SHA384:" \
-                                   "TLS_CHACHA20_POLY1305_SHA256:" \
-                                   "TLS_AES_128_GCM_SHA256"
-# else
-#  define TLS_DEFAULT_CIPHERSUITES "TLS_AES_256_GCM_SHA384:" \
+# if OPENSSL_API_LEVEL < 4
```

Comment:
Should this be:

`# if !OPENSSL_API_3`

Reply:
@levitte and @kroeckx said it should be

`# if OPENSSL_API_LEVEL < 4` 

in the previous comment (now resolved)

Reply:
I believe this is still wrong. There are no other instances of this pattern to deprecate stuff. Everywhere else we use `# if !OPENSSL_API_3`

Reply:
I agree with matt.

Reply:
I'll make the change.